### PR TITLE
Stabilizing cypress tests

### PR DESCRIPTION
**Pull request by MorganLove**
**Description**

- Upgraded Cypress to take advantage of `cy.intercept`
- Refined tests to make them stable

### CODE REVIEW

**Comment by MorganLove** [comment]
Huh, I didn't get either of those errors locally. * investigating *

**Comment by MorganLove** [comment]
Okay, it looks like this is stable now. Yey!

**Review by jonahkagan — APPROVED** [review]
nice improvements!

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  cypress: cypress-io/cypress@1
+  cypress: cypress-io/cypress@1.27.0
```
[the remark this answers is not in the document]
Comment (MorganLove) [reply .circleci/config.yml:3]: There was an issue with the orb trying to install two versions of cypress simultaneously and causing weird problems, and I was trying to fix that. This was part of my troubleshooting. I am actually not sure if it contributed to it working finally or not -- I'll take it out and see what happens.

**Pull request merged**